examples/djangosite/djangosite/settings.py

```diff
@@ -128,3 +128,36 @@
 # https://docs.djangoproject.com/en/4.1/ref/settings/#default-auto-field
 
 DEFAULT_AUTO_FIELD = "django.db.models.BigAutoField"
+
+NGROK_CONFIG = {
+    "policies": {
```

Comment:
We could expose the ability to set other config settings (such as metadata) here.